benchmarks/profile.py

```diff
@@ -0,0 +1,228 @@
+import json
+import os
+import pathlib
+import re
+import sys
+import warnings
+
+import jax.numpy as jnp
+import matplotlib.pyplot as plt  # pyright: ignore[reportMissingImports]
+import numpy as np  # pyright: ignore[reportMissingImports]
+import pandas as pd  # pyright: ignore[reportMissingImports]
```

Comment:
Is pandas that useful here? (It seems fairly minimal below.)

To me this is in the same bucket as libraries like huggingface and lightning - reasonable for hacking around but not a thing I'd like to check in. (I realise this is a somewhat unusual position.)